GameData/ROCapsules/PartConfigs/Soyuz MS/SoyuzMSOrbitalModule.cfg

```diff
@@ -0,0 +1,219 @@
+	//	Model and Dimensions
+	//	============================================================================
+
+	MODEL
+	{
+		model = ROCapsules/Assets/DECQ/SoyuzMS/BO
+	}
+
+	scale = 1.0
+	rescaleFactor = 1.0
+
+	node_stack_top = 0.0, 1.243, 0.0, 0.0, 1.0, 0.0, 2
+	node_stack_bottom = 0.0, -1.032, 0.0, 0.0, -1.0, 0.0, 2
+	node_attach = 0.0, 0.0, 0.0, 0.0, -1.0, 0.0
+
+	attachRules = 1,0,1,1,0
+
+	//	============================================================================
+	//	Title, Description, Category, Techs
+	//	============================================================================
+
+	TechRequired = unlockParts
+	cost = 150
+	entryCost = 1
+	category = Pods
+	subcategory = 0
+
+	title = Soyuz TM/TMA/MS Orbital Module
+	manufacturer = RKK Energia
+	description = The Orbital Module (BO) of the Soyuz contains everything that it is not needed for reentry. This includes the docking system, the airlock and the KURS automatic docking antennas. It is jettisoned before entering the atmosphere.
+	tags = soyuz ms, DECQ
+
+	bulkheadProfiles = srf, size2
+
+	//	============================================================================
+	//	Thermo, Durability
+	//	============================================================================
+
+	mass = 1.05
+	dragModelType = default
+	maximum_drag = 0.25
+	minimum_drag = 0.25
+	angularDrag = 0.5
+	crashTolerance = 10
+	breakingForce = 250
+	breakingTorque = 250
+	maxTemp = 673.15
+	skinMaxTemp = 673.15
+	stagingIcon = DECOUPLER_HOR
+	stageOffset = 1
+	childStageOffset = 1
+
+	//	============================================================================
+	//	Modules and Resources
+	//	============================================================================
+
+	MODULE
+	{
+		name = ModuleDecouple
+		menuName = Jettison Orbital Module
+		ejectionForce = 100
+		isOmniDecoupler = False
+		explosiveNodeID = bottom
+	}
+
+	MODULE
+	{
+		name = ModuleAnimateGeneric
+		animationName = ANTENNA
+		isOneShot = False
+		allowManualControl = True
+		startEventGUIName = Deploy KURS Antenna
+		endEventGUIName = Retract KURS Antenna
+		actionGUIName = Toggle KURS Antenna
+		allowAnimationWhileShielded = False
+	}
+
+	MODULE
+	{
+		name = ModuleLight
+		lightName = light
+		useAnimationDim = True
+		lightBrightenSpeed = 9
+		lightDimSpeed = 9
+		animationName = LIGHT_SOYUZ
+		resourceAmount = 0.075
+		useResources = true
+	}
+
+	MODULE:NEEDS[ConnectedLivingSpace]
+	{
+		name = ModuleConnectedLivingSpace
+		passable = True
+		passableWhenSurfaceAttached = False
+		surfaceAttachmentsPassable = False
+		impassablenodes = F1, F2
+	}
+}
+
+//	==================================================
+//	TAC Life Support compatibility.
+//	==================================================
+
+@PART[ROC-SoyuzMSOM]:FOR[RealismOverhaul]:NEEDS[TacLifeSupport]
+{
+	@description ^= :$: Provides support resources for a crew of 3 for up to 30 days.
+
+	@mass -= 0.315
```

Comment:
Don't subtract out the mass of the resources from the part.  Just let it be empty space if needed.